mantidproject/mslice
# Allow users generate scripts from generated scripts

Description of work.
Default plot options for cuts and slices and now saved when their respective windows are shown. This allows the generated scripts to know what changes have been made to the plot since it was loaded.
**To test:**
Open Mslice. Plot a slice or cut and generate a script.
Run the generated script, make changes and generate another script.
Observe the difference between the first and second generated script.



Fixes #420 .
